topgun/k8s/external_postgres_test.go

```diff
@@ -0,0 +1,56 @@
+package k8s_test
+
+import (
+	"fmt"
+	"path"
+
+	. "github.com/concourse/concourse/topgun"
+	. "github.com/onsi/ginkgo"
+)
+
+var _ = Describe("External PostgreSQL", func() {
+	var (
+		releaseName   string
+		pgReleaseName string
+		namespace     string
+	)
+
+	BeforeEach(func() {
+		releaseName = fmt.Sprintf("topgun-ep-%d-%d", GinkgoRandomSeed(), GinkgoParallelNode())
+		namespace = releaseName
+		pgReleaseName = releaseName + "-pg"
+
+		helmDeploy(pgReleaseName,
+			namespace,
+			path.Join(Environment.ChartsDir, "stable/postgresql"),
+			"--set=livenessProbe.initialDelaySeconds=3",
+			"--set=livenessProbe.periodSeconds=3",
+			"--set=persistence.enabled=false",
+			"--set=postgresqlDatabase=pg-database",
+			"--set=postgresqlPassword=pg-password",
+			"--set=postgresqlUsername=pg-user",
+			"--set=readinessProbe.initialDelaySeconds=3",
+			"--set=readinessProbe.periodSeconds=3",
+		)
+
+		deployConcourseChart(releaseName,
```

Comment:
might not be super relevant, should we move this part to the suite, since that is the basic deploy that we do in almost every test?

Reply:
Hey, what do you mean by `this part`? The `releaseName` only, or the other settings?

Reply:
I meant deploying concourse chart.
my thought was that we create a `justbeforeeach` in the suite that deploys concourse chart using whatever params that we set here.

does this make any sense?

Reply:
hmmmm I see; my initial thoughts were more around not having too many things global (in `k8s_suite.go`) to avoid having to know what are the right variables to set when creating a new test 🤔 

Would you mind leaving that discussion to a separate issue? There we could elaborate more on it.

thanks!!